Lib/tempfile.py

```diff
@@ -151,9 +151,7 @@ def __iter__(self):
         return self
 
     def __next__(self):
-        c = self.characters
-        choose = self.rng.choice
-        letters = [choose(c) for dummy in range(8)]
+        letters = self.rng.choices(self.characters, k=8)
```

Comment:
I don't think these kinds of trivial changes should be done.  There should be a preference for code stability.